WordPress/Classes/WordPress.xcdatamodeld/WordPress 87.xcdatamodel/contents

```diff
@@ -740,6 +740,10 @@
         <attribute name="slug" attributeType="String" syncable="YES"/>
         <relationship name="blog" maxCount="1" deletionRule="Nullify" destinationEntity="Blog" inverseName="roles" inverseEntity="Blog" syncable="YES"/>
     </entity>
+    <entity name="SearchResultsStatsRecordValue" representedClassName=".SearchResultsStatsRecordValue" parentEntity="StatsRecordValue" syncable="YES">
+        <attribute name="searchTerm" optional="YES" attributeType="String" syncable="YES"/>
```

Comment:
I see that `searchTerm` is designated "optional" for both Core Data and in Swift here.

To make sure I'm clear after our discussion in #11010, that's okay because in this case, it's okay to proceed to save `nil` for `searchTerm`. Correct?

Reply:
Yup! 

I _think_ I'll change it later to force it to not be optional, but I haven't played with the backend service to see whether it can return empty results, so I wanted to play it safe here.